src/messaging/inbound/data-ingest/processors/file-metadata.js

```diff
@@ -21,7 +21,6 @@ const processV1FileMetadata = async (message) => {
     })
   }
   await persistFileMetadata(validated.metadata)
```

Comment:
Do we still need to await this, when the next line of action has been moved?

Reply:
yes, it doesnt handle the error correctly otherwise